src/routes/entry.js

```diff
@@ -52,10 +73,63 @@ module.exports = function (hds) {
         this.body = entry;
     }
 
-    function*getAttachment() {
+    function* createEntry() {
+        var data = this.request.body;
+        try {
+            var value = yield hds.Entry.create(this.params.kind, data, {owner: 'test@cheminfo.org'}).save();
```

Comment:
@targos @lpatiny @andcastillo when we did the document we forgot to specify the owner part, and thats mandatory for the creation of an entry, any idea of how to solve it?